### PR TITLE
Added MiniBatchKmeans from ClusterR package to Learners

### PR DESCRIPTION
**Pull request by Prasiddhi**
Added MiniBatchKmeans from ClusterR package.

Prediction is via predict_MBatchKMeans of ClusterR package.

We are always happy to receive pull requests.

Please make sure you have read our coding guidelines: 
https://github.com/mlr-org/mlr/wiki/mlr-Coding-Guidelines

This especially means that you have understood:

* The style guide - our lintr will provide you feedback on this
* How to run tests locally. Yes, travis will also run them for you, bad it is annoying to wait for this.
* How to run R CMD CHECK locally. See point before. 

Also it's helpful to get into direct contact with the suggested reviewers to get help, getting your PR merged.
You might want to join our slack at: 
https://mlr-org.slack.com

### CODE REVIEW

File: tests/testthat/test_cluster_MiniBatchKmeans.R [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  )
+
+  old.predicts.list = list()
+
+  for (i in seq_along(parset.list)) {
+    parset = parset.list[[i]]
+    set.seed(getOption("mlr.debug.seed"))
+    m = ClusterR::MiniBatchKmeans(noclass.train, clusters = 2L, batch_size = 10L)
+    p = as.integer(ClusterR::predict_MBatchKMeans(data = noclass.test, CENTROIDS = m$centroids, fuzzy = FALSE))
+    p[p == 0] = NA
```
Comment (larskotthoff) [inline tests/testthat/test_cluster_MiniBatchKmeans.R]: Why are you setting predictions to NA here?

File: tests/testthat/test_cluster_MiniBatchKmeans.R [hunk 2]
```diff
@@ -0,0 +1,24 @@
+context("cluster_MiniBatchKmeans")
+
+test_that("cluster_MiniBatchKmeans", {
+  requirePackagesOrSkip("ClusterR", default.method = "load")
+
+  parset.list = list(
+    list()
```
Comment (larskotthoff) [inline tests/testthat/test_cluster_MiniBatchKmeans.R]: Can you add another parameter setting to test please?

File: tests/testthat/test_cluster_MiniBatchKmeans.R (hunk 1 above)
Comment (Prasiddhi) [reply tests/testthat/test_cluster_MiniBatchKmeans.R]: Sorry, I was testing whether ClusterR returns 0 as a cluster index. Removed from code.

File: tests/testthat/test_cluster_MiniBatchKmeans.R (hunk 2 above)
Comment (Prasiddhi) [reply tests/testthat/test_cluster_MiniBatchKmeans.R]: Added one more test setting.

**Comment by pat-s** [comment]
@Prasiddhi 
Thanks for contributing. Could you please add the package to DESCRIPTION and an entry to NEWS.md, following the structure you find in there?

**Comment by pat-s** [comment]
@larskotthoff If you approve, please merge.

**Comment by larskotthoff** [comment]
Thanks, merging.

**Pull request merged**